src/test/java/bio/terra/service/duos/DuosDaoTest.java

```diff
@@ -107,13 +120,37 @@ public void testInsertFirecloudGroupThrowsOnDuplicateDuosId() {
                     .firecloudGroupEmail("another-fc-group-name@dev.test.firecloud.org")));
   }
 
-  private void verifyRetrievedFirecloudGroupContents(DuosFirecloudGroupModel retrieved) {
+  @Test
+  public void testUpdateFirecloudGroupLastSyncedDateForNonexistentRow() {
+    assertFalse(duosDao.updateFirecloudGroupLastSyncedDate(UUID.randomUUID(), Instant.now()));
+  }
+
+  @Test
+  public void testUpdateFirecloudGroupLastSyncedDate() {
+    duosFirecloudGroupId = duosDao.insertFirecloudGroup(toInsert);
+    Instant lastSyncedDate = Instant.parse("2022-11-17T00:00:00.00Z");
```

Comment:
Is there a reason to not use `now()` here?

Reply:
It looks like it's needed because the database truncates the timestamp (so it's really a date stamp, not a time stamp). If so, I think it would be better to truncate in the test instead of hardcoding a value. I think `Instant.truncatedTo(ChronoUnit.DAYS)` could do this.

Looking at the code, I don't see any use of passing in an explicit time to `updateFirecloudGroupLastSyncedDate()` except in this test. IMO it would be better to remove this argument and use some other way to validate the updated date. One way would be to save the value of "now" before running the sync, and assert that the sync time is after the saved time.